cms/core/wagtail_hooks.py

```diff
@@ -28,5 +28,10 @@ def editor_js() -> str:
     return format_html('<script src="{}"></script>', static("js/wagtail-editor-customisations.js"))
 
 
+@hooks.register("insert_global_admin_css")
+def global_admin_css() -> str:
+    return format_html('<link rel="stylesheet" href="{}">', static("css/admin.css"))
```

Comment:
Does this require a local config update to work? Since pulling this change down I'm getting errors trying to access the admin site

```
ValueError: Missing staticfiles manifest entry for 'css/admin.css'
```

Reply:
Of course, thanks, haven't had to run that for a while.